### DISCUSSION TITLE
Child Sync Policy to Parent Sync Policy in Apps of Apps Pattern

### DISCUSSION THREAD

**Discussion by tlmanz** [argoproj/argo-cd#20535] [Q&A]
Is it possible to set the Child Sync Policy to inherit the Parent Sync Policy within an Apps of Apps pattern? Specifically, if the Parent App is configured with a certain sync policy, can this be automatically applied to all child applications, or do we need to configure each child individually? Additionally, if inheritance is possible, what is the recommended approach or configuration to achieve this setup effectively?

Idea is to change the sync policy of the Parent via UI and it will affect the child sync policies for easy management

**Comment by kostis-codefresh** [comment]
> Additionally, if inheritance is possible, what is the recommended approach or configuration to achieve this setup effectively?

Have you seen ApplicationSets?
